$NetBSD: patch-ae,v 1.24 2025/04/24 13:35:29 gdt Exp $

- Use 'su -' instead of just 'su'. Fixes priv delegation on SunOS
  and AIX where Erlang relies on $HOME set correctly. Use a portable
  su arg order while at it.

\todo File upstream.

--- ejabberdctl.template.orig	2023-01-17 10:35:52.000000000 +0000
+++ ejabberdctl.template
@@ -121,7 +121,7 @@ set_dist_client()
 exec_cmd()
 {
     case $EXEC_CMD in
-        as_install_user) su -s /bin/sh -c '"$0" "$@"' "$INSTALLUSER" -- "$@" ;;
+        as_install_user) su - "$INSTALLUSER" -c '"$@"' -- "$@" ;;
         as_current_user) "$@" ;;
     esac
 }
